packages/core/src/metrics/utils.ts

```diff
@@ -0,0 +1,57 @@
+import type { MeasurementUnit, MetricBucketItem } from '@sentry/types';
+import { dropUndefinedKeys } from '@sentry/utils';
+import type { MetricType, SimpleMetricBucket } from './types';
+
+/**
+ * Generate bucket key from metric properties.
+ */
+export function getBucketKey(
+  metricType: MetricType,
+  name: string,
+  unit: MeasurementUnit,
+  tags: Record<string, string>,
+): string {
+  const stringifiedTags = Object.entries(dropUndefinedKeys(tags)).sort() as unknown as string[];
+  return [metricType, name, unit].concat(stringifiedTags).join('');
```

Comment:
This creates an intermediary array. We don't need it.

Reply:
you're right, will fix